site/src/components/SignupForm/signupform.style.js

```diff
@@ -1,83 +1,101 @@
 import styled from "styled-components";
 
 const SignupFormWrapper = styled.section`
-
+.Form-page{
+	display:flex;
+	margin: 0 4rem;
+	@media only screen and (max-width: 900px){
+		display: block;
+	}
+}
 	.required-sign {
 		color: #8B0000;
 	}
-
 	.desc-text {
-		margin: 1rem auto;
+		margin: 1rem;
+		p{
+		font-size: 1.35rem;
+		}
 	}
-
   .accounts{
+	  margin-top: 1rem;
     p {
-			font-size: 0.9rem;
+			font-size: 0.8rem;
 			line-height: 0.9rem;
 			color: #aaa;
     }
     .accounts_group{
       margin: 0 1rem 0 1rem;
     }
-  }
+  } 
+  .form-container{
+	  width: 170rem;
+	  margin: 1rem;
+       
+	  @media only screen and (max-width: 1000px){
+		width: 100%;
+		margin: 2rem 0 ;
+	}
 
-	.form {
+  .form-title {
+	padding: 1rem;
+	text-align: center;
+	background-color: #00b39f;
+	color: white;
+	width: 100%;
+}
+	.form1 {
 		font-size: .8725rem;
-		margin: 0rem 0rem 0rem 0rem;
+		box-shadow: 0px 10px 10px 2px rgba(30, 33, 23, 0.5);
```

Comment:
The `box-shadow` should be around the complete `form`. Currently it is not around the title of the form.